### PR TITLE
Created README.MD

### PR DESCRIPTION
**Pull request by luis-gustavo**
# Related Issues

#3 

# Description

Created a README.MD with all the useful informations about the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+| Luis Gustavo Avelino | [luis-gustavo](https://github.com/luis-gustavo) | luis.gustavo3013@hotmail.com |
+| Matheus Felizola     | [Felizolinha](https://github.com/Felizolinha)   | matheusbsb2009@Hotmail.com |
+
+# Usage
+
+If you are interested in using this project just fork you clone this repository.
+
+## Technologies
+
+The neural network is build and trained using [Anaconda](https://www.anaconda.com/) and [Python3](https://www.python.org/downloads/).
```
Comment (Felizolinha) [inline README.md]:
```suggestion
The neural network is built and trained using [Tensorflow](https://www.tensorflow.org/) and [Python3](https://www.python.org/downloads/).
```

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+This project is led by [Luis Gustavo]()
+
+| Member               | github        | email                  |
+|----------------------|--------------|------------------------|
+| Luis Gustavo Avelino | [luis-gustavo](https://github.com/luis-gustavo) | luis.gustavo3013@hotmail.com |
+| Matheus Felizola     | [Felizolinha](https://github.com/Felizolinha)   | matheusbsb2009@Hotmail.com |
+
+# Usage
+
+If you are interested in using this project just fork you clone this repository.
```
Comment (Felizolinha) [inline README.md]:
```suggestion
If you are interested in using this project just fork or clone this repository.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+
+# About
+
+American Sign Language (ASL) is a complete, natural language that has the same linguistic properties as spoken languages, with grammar that differs from English. ASL is expressed by movements of the hands and face. It is the primary language of many North Americans who are deaf and hard of hearing, and is used by many hearing people as well.
+
+Reliable estimates for American ASL users range from 250.000 to 500.000 persons, including a number of children of deaf adults.
+
+## Solution
+
+The purpose of this project is to build a tool that will correctly predict the digit from an image, this tool can be used in a iOS device. The tool should be able to predict the digit independently of the skin or the background color.
```
Comment (Felizolinha) [inline README.md]:
```suggestion
The purpose of this project is to build a tool that will correctly predict the digit from an image, this tool can be used on an iOS device. The tool should be able to predict the digit independently of skin or background color.
```

**Comment by luis-gustavo** [comment]
@Felizolinha All of the changes were made.

**Review by Felizolinha — APPROVED** [review]

**Pull request merged**